calibr8/optimization.py

```diff
@@ -156,6 +156,104 @@ def fit_scipy(
     return fit.x, history
 
 
+def fit_scipy_global(
+    model: core.CalibrationModel,
+    *,
+    independent: numpy.ndarray,
+    dependent: numpy.ndarray,
+    theta_bounds: list,
+    method: str = None,
+    maxiter: int = 5000,
+    minimizer_kwargs: dict = None,
+):
+    """Function to fit the calibration model with observed data using global optimization.
+
+    Parameters
+    ----------
+    model : calibr8.CalibrationModel
+        the calibration model to fit (inplace)
+    independent : array-like
+        desired values of the independent variable or measured values of the same
+    dependent : array-like
+        observations of dependent variable
+    theta_bounds : array-like
+        bounds to fit the parameters
+    method: str, optional
+        Type of solver. Should be
+            - 'dual_annealing'         : ref:`(see here) <optimize.dual_annealing>`
+        If not given, chosen to be ``dual_annealing``.
```

Comment:
Why making this optional is there is only one option? Do you want to keep the method flesxibel for the future?
For now, this kwarg is unnecessary.

Reply:
There was support for multiple solvers just two commits ago. However, I couldn't produce a scenario where these alternative solvers (scipy global opt alternatives) outperformed dual_annealing.
That does not mean that such a scenario is not possible.
Either way, global optimization is a particular topic. I plead for expanding functionality by providing different solvers that users can choose from at their discretion.